app/models/miq_provision_orch_workflow.rb

```diff
@@ -0,0 +1,66 @@
+class MiqProvisionOrchWorkflow < MiqProvisionVirtWorkflow
+  def initialize(values, requester, options = {})
+    initial_pass = values.blank?
+    initial_pass = true if options[:initial_pass] == true
+    instance_var_init(values, requester, options)
+
+    # Check if the caller passed the source VM as part of the initial call
+    if initial_pass == true
+      src_vm_id = get_value(@values[:src_vm_id])
+      unless src_vm_id.blank?
+        vm = OrchestrationTemplate.find_by(:id => src_vm_id)
```

Comment:
Since this isn't a VM can we rename the `vm` var?  Does the `@values` still need to be `src_vm_id`?

Reply:
`src_vm_id` is heavily used and relied on in the code.